.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 ## Description
 
-Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../CONTRIBUTIONS.md).
+Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../blob/1.x/CONTRIBUTIONS.md).
```

Comment:
Why we need to update this link? Should not it always be pointing to default branch?

Reply:
Clicking on the before contribution link above, as well as when [contributors](https://github.com/ChainSafe/web3.js/pull/4864#discussion_r830292670) make PRs leads to a [404](https://github.com/ChainSafe/web3.js/CONTRIBUTIONS.md)

Reply:
@luu-alex @nazarhussain Thanks that's reason.

Reply:
I think this PR is fine, but we may need a different PR template for 4.x